packages/node/src/integrations/undici/index.ts

```diff
@@ -44,6 +46,13 @@ export interface UndiciOptions {
    * Defaults to true
    */
   breadcrumbs: boolean;
+
+  /**
+   * Whether tracing spans should be created for requests
+   * If not set, this will be enabled/disabled based on if tracing is enabled.
+   */
+  tracing?: boolean;
```

Comment:
l: I'd slightly reword this based on my understanding that this option not only controls span creation but also distributed tracing:


```suggestion
  /**
   * Controls whether tracing spans should be created for requests and if 
   * distributed tracing headers should be attached to outgoing requests.
   *
   * @default If not set, tracing follows global SDK settings (`tracesSampleRate`).
   */
  tracing?: boolean;
```